src/openapi/openapi-service.ts

```diff
@@ -0,0 +1,46 @@
+import { IProxyConfig } from '../config';
+import { RequestHandler, Application } from 'express';
+import { OpenAPIV3 } from 'openapi-types';
+// @ts-expect-error
+import openapi from '@unleash/express-openapi';
+import { createOpenApiSchema } from './.';
+
+// Partial types for "@unleash/express-openapi".
+interface IExpressOpenApi extends RequestHandler {
```

Comment:
Would be nice if this matched the backend. Slight changes, and moved to a `d.ts` file:

https://github.com/Unleash/unleash/blob/main/src/lib/types/openapi.d.ts

Should allow us to remove the above `// @ts-expect-error`.

Reply:
Hmm, I've moved the file, but I still get an error when I try to compile without the `@ts-expect-error` directive. Do you know what that's about? The changes are in 1d6f0bf5c4c241b7bb9e7117f7b100cbab659411.

Reply:
Interestingly it works correctly in tests (and it complains that the directive isn't needed)